config/replaceobj.go

```diff
@@ -59,7 +59,7 @@ func jsonCompatTypes(val interface{}) error {
 				return err
 			}
 		}
-	case string, float64, bool, nil:
+	case string, float64, bool, int, nil:
```

Comment:
Sorry, I've formalized config structure to be: string, float64, bool, nil, map[string]interface{}, and []interface{} types.
Checkout: https://github.com/taskcluster/taskcluster-worker/pull/125

I fixed things there so YAML doesn't unmarshal integers to int, but to float64, like json would...